src/index.ts

```diff
@@ -1 +1 @@
-console.info("hello");
\ No newline at end of file
+console.info("hey"); // nice, now what?
```

Comment:
```suggestion
console.info("hey"); 
```
the comment shouldn't be there